pages/index.js

```diff
@@ -21,19 +20,17 @@ export default function Home({allPostsData}) {
 
       </Head>        
 
-      <section className={utilStyles.headingMd}>
-        <p>Student of Computer and Data Sciences with a specialization in Economics. I have a strong interest in the intersection of technology and business.</p>
-        <small>PS : Coffee, travel and gym addict</small>
-      </section> 
+      <section className="space-y-4 font-mono text-justify">
+        <p className="text-lg">Student of Computer and Data Sciences with a specialization in Economics. I have a strong interest in the intersection of technology and business.</p>
+        <small className='text-xs'>PS : Coffee, travel and gym addict</small>
```

Comment:
Don't mix single and double quotes for the attributes for example. See a linter like to eslint to prevent this.